tests/verkle/eip4762_verkle_gas_witness/test_contract_execution.py

```diff
@@ -52,157 +52,175 @@ def code_with_jumps(size, jumps: list[Jump | Jumpi] = []):
+        #     bytes(
+        #         Op.PUSH0 * 10
+        #         + Op.JUMP(10 + 3 + 1 + 1000 + 1)  # 3+1=PUSH2+JUMP
+        #         + Op.PUSH0 * 1000
+        #         + Op.PUSH1(0x5B)
+        #         + Op.PUSH0 * 100  # Add more code, but can't be executed due to the invalid jump
+        #     ),
+        #     1_000_000,
+        #     [[0, 0], [32, 32]],
+        # ),
+        # (  # jumpi_to_jumpdest_in_pushn_data
+        #     bytes(
+        #         Op.PUSH0 * 10
+        #         + Op.JUMPI(10 + 5 + 1 + 1000 + 1, 1)  # 5+1=PUSH1+PUSH2+JUMPI
+        #         + Op.PUSH0 * 1000
+        #         + Op.PUSH1(0x5B)
+        #         + Op.PUSH0 * 100
+        #     ),
+        #     1_000_000,
+        #     [[0, 0], [32, 32]],
+        # ),
+        # (  # jump_to_non_jumpdest_destiny
+        #     bytes(
+        #         Op.PUSH0 * 10 + Op.JUMP(10 + 3 + 1 + 1000) + Op.PUSH0 * 1000 + Op.ORIGIN
+        #     ),  # 3+1=PUSH2+JUMP
+        #     1_000_000,
+        #     [[0, 0], [32, 32]],
+        # ),
+        # (  # jumpi_to_non_jumpdest_destiny
+        #     bytes(
+        #         Op.PUSH0 * 10 + Op.JUMPI(10 + 5 + 1 + 1000, 1) + Op.PUSH0 * 1000 + Op.ORIGIN
+        #     ),  # 5+1=PUSH1+PUSH2+JUMPI
+        #     1_000_000,
+        #     [[0, 0], [32, 32]],
+        # ),
+        # (  # linear_execution_stopping_at_first_byte_of_next_chunk
+        #     code_with_jumps(128 * 31 + 1),
+        #     1_000_000,
+        #     [[0, 128]],
+        # ),
+        # (  # false_jumpi
+        #     code_with_jumps(150 * 31 + 10, [Jumpi(50, 1000, False)]),
+        #     1_000_000,
+        #     [[0, 150]],
+        # ),
+        # (  # insufficient_gas_for_jump_instruction
+        #     code_with_jumps(150 * 31, [Jump(10, 1000)]),
+        #     21000 + 200 + 10 + 3,
+        #     [[0, 0]],
+        # ),
+        # (  # insufficient_gas_for_jumpi_instruction
+        #     code_with_jumps(150 * 31, [Jumpi(10, 1000, True)]),
+        #     21000 + 200 + 10 + 3 + 3,
+        #     [[0, 0]],
+        # ),
+        # (  # sufficient_gas_for_jump_instruction_but_not_for_code_chunk
+        #     code_with_jumps(150 * 31, [Jump(10, 1000)]),
+        #     21000 + 200 + 10 + 3 + 8,
+        #     [[0, 0]],
+        # ),
+        # (  # sufficient_gas_for_jumpi_instruction_but_not_for_code_chunk
+        #     code_with_jumps(150 * 31, [Jumpi(10, 1000, True)]),
+        #     21000 + 200 + 10 + 3 + 3 + 10,
+        #     [[0, 0]],
+        # ),
+        # (  # jump_outside_code_size
+        #     code_with_jumps(150 * 31, [Jump(10, 150 * 31 + 42)]),
+        #     1_000_000,
+        #     [[0, 0]],
+        # ),
+        # (  # jumpi_outside_code_size
+        #     code_with_jumps(150 * 31, [Jumpi(50, 150 * 31 + 42, True)]),
+        #     1_000_000,
+        #     [[0, 0]],
+        # ),
         (  # push20 with data split in two chunks
             Op.PUSH0 * (31 - (1 + 10)) + Op.PUSH20(0xAA),
```

Comment:
This is actually testing an interesting case about code-chunking that might have actually triggered a bug in the Python impl.

There's a `PUSH20` which spans 2 code-chunks.